applications/MeshMovingApplication/custom_utilities/fixed_mesh_ale_utilities.cpp

```diff
@@ -192,13 +194,24 @@ namespace Kratos
 
         // Search the origin model part nodes in the virtual mesh elements and
         // interpolate the values in the virtual element to the origin model part node
-        block_for_each( rOriginModelPart.Nodes(),
-            [&]( Node<3>& rNode )
+
+        unsigned int max_results=10000;
+        double search_tol = 1e-5;
+        block_for_each( rOriginModelPart.Nodes(), typename BinBasedFastPointLocator<TDim>::ResultContainerType(max_results),
+            [&]( Node<3>& rNode,
+                 typename BinBasedFastPointLocator<TDim>::ResultContainerType& search_results
+             )
             {
                 // Find the origin model part node in the virtual mesh
-                Vector aux_N;
+                Vector aux_N(4);
                 Element::Pointer p_elem = nullptr;
-                const bool is_found = bin_based_point_locator.FindPointOnMeshSimplified(rNode.Coordinates(), aux_N, p_elem);
+                const bool is_found = bin_based_point_locator.FindPointOnMesh(
```

Comment:
This is faster than FindPointOnMeshSimplified?

Reply:
yes, because it avoids creating the temporary array "search results"

Reply:
Good to know, because there are two processes in the MeshingApplication using this